src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_VALUE = 10.0;
+    private static final int MULTIPLAYER = 100;
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+
+    public FigureSupplier() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Figure getRandomFigure() {
+        FiguresShape[] figureshape = FiguresShape.values();
+        Figure fg = null;
```

Comment:
Is this variable really necessary? I'd recommend return figure directly, without this var usage